webservices/partition/sched_a.py

```diff
@@ -40,6 +42,7 @@ def column_factory(cls, parent):
                 parent.c.contbr_id,
                 parent.c.cmte_id,
             ).label('clean_contbr_id'),
+            sa.cast(sa.func.get_transaction_year(parent.c[cls.transaction_date_column], parent.c.rpt_yr), sa.Numeric(4, 0)).label('transaction_two_year_period'),
```

Comment:
Without the `cast`, the `select` query that gets generated as a part of the `create` statements for the child tables contains this at the end:

```
create table ofec_sched_b_2015_2016_tmp as SELECT sched_b.sched_b_sk, sched_b.form_tp, sched_b.cmte_id, ..., get_transaction_year(sched_b.disb_dt, sched_b.rpt_yr) AS transaction_two_year_period
```

This results in the error above, which doesn't make sense to me given that the column structure itself is defined as `Numeric(4, 0)` via sqlalchemy.  Sure enough, if you look at the master table you see that the `transaction_two_year_period` column is of type `numeric(4, 0)`, but in the child table you only see it as `numeric`.

If I provide the `cast` though, then the query generated looks like this:

```
create table ofec_sched_b_2015_2016_tmp as SELECT sched_b.sched_b_sk, sched_b.form_tp, sched_b.cmte_id, ..., CAST(get_transaction_year(sched_b.disb_dt, sched_b.rpt_yr) AS NUMERIC(4, 0)) AS transaction_two_year_period
```

In this case the column is created correctly in the child tables with the matching `numeric(4, 0)` type.